benchmarks/compare.py

```diff
@@ -64,14 +61,9 @@ def load_from(cls, data: Dict[str, Any]) -> QueryRun:
     def execution_time(self) -> float:
         assert len(self.iterations) >= 1
 
-        # If we don't have enough samples, median() is probably
-        # going to be a worse measure than just an average.
-        if len(self.iterations) < MEAN_THRESHOLD:
-            method = statistics.mean
-        else:
-            method = statistics.median
-
-        return method(iteration.elapsed for iteration in self.iterations)
+        # Use minimum execution time to account for variations / other
+        # things the system was doing
+        return min(iteration.elapsed for iteration in self.iterations)
```

Comment:
lgtm, but imho, can `min` be deceptive comparing to `avg/mean/median`? 🤔

Reply:
I agree it is misleading -- in terms of measuring a change between datafusion versions, I think `min` will give us the least variance between runs and represents best case performance.

However, it doesn't really give a sense for how much variation is across runs 🤔